main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"log"
+
+	"context"
+
+	"github.com/shomali11/slacker"
+)
+
+func main() {
+	var config Config
+	err := config.FromEnv()
```

Comment:
use our standard kingpin CLI libarary here please

Reply:
TBD in future. Deferred now for time contrains.